File: 1742-widest-vertical-area-between-two-points-containing-no-points/widest-vertical-area-between-two-points-containing-no-points.java
class Solution {
    public int maxWidthOfVerticalArea(int[][] points) {
        Arrays.sort(points, (a,b) -> Integer.compare(a[0], b[0]));
        int ans = -1;
        for(int i=0; i<points.length-1; i++) {
            ans = Math.max(ans, points[i+1][0] - points[i][0]);
        }
        return ans;
    }
}

File: 1742-widest-vertical-area-between-two-points-containing-no-points/widest-vertical-area-between-two-points-containing-no-points.js
/**
 * @param {number[][]} points
 * @return {number}
 */
var maxWidthOfVerticalArea = function(points) {
    points.sort((a, b) => a[0] - b[0]);
    let ans = -1;
    for(let i=0; i<points.length-1; i++) {
        ans = Math.max(ans, points[i+1][0] - points[i][0]);
    }
    return ans;
};